home/views.py

```diff
@@ -14,17 +16,35 @@ def landingpage(request):
 
 def displayQuestion(request, **kwargs):
     question = Question.objects.all().get(id=kwargs['pk'])
-    sortAnsBy = request.GET["sortanswersby"] if 'sortanswersby' in request.GET else ''
+    sort_answer_by = request.GET["sortanswersby"] if 'sortanswersby' in request.GET else ''
```

Comment:
I think instead of doing if statement  you can just write here the following line
` sort_answer_by = request.GET["sortanswersby", '']`

Reply:
Thanks I fixed it